src/styles/block/card.scss

```
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 6px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    width: 100%;
+    margin-bottom: 19px;
+    font-size: 10px;
+    font-weight: 400;
+    color: $color-secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    width: 100%;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 23px;
+
+    &-text {
+      display: block;
+      font-size: 10px;
+      line-height: 15px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 100%;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-secondary;
+    }
+
+    &-number {
+      font-size: 16px;
+      line-height: 19px;
+      font-weight: 700;
+      color: $color-main-accent;
+    }
+  }
+
+  &__button {
+    width: 166px;
```

no needs to fix width for your button as you have paddings and fixed width for your card
```suggestion
    width: 100%;
```